WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -50,6 +50,9 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
 
     func removeItemFromCart(_ cartItem: CartItem) {
         itemsInCart.removeAll(where: { $0.id == cartItem.id })
+        if itemsInCart.isEmpty {
```

Comment:
Agree, it does makes sense to switch back. 

Should we more explicit about it? Since right now it's hidden on implementation details as a side-effect of removing "_items from cart until is empty_", perhaps something simple like:

```suggestion
      checkIfCartEmpty()
  }
        
  private func checkIfCartEmpty() {
    if itemsInCart.isEmpty {
      orderStage = .building
    }
  }

```